src/alias/azext_alias/telemetry/telemetry.py

```diff
@@ -0,0 +1,127 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
```

Comment:
Why not just have `src/alias/azext_alias/telemetry.py`?
Seems like you only have one file under `src/alias/azext_alias/telemetry`.